npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -34,14 +34,10 @@ const (
 )
 
 type IPSetManager struct {
-	iMgrCfg *IPSetManagerCfg
-	setMap  map[string]*IPSet
-	// Map with Key as IPSet name to to emulate set
-	// and value as struct{} for minimal memory consumption.
-	toAddOrUpdateCache map[string]struct{}
-	// IPSets referred to in this cache may be in the setMap, but must be deleted from the kernel
-	toDeleteCache map[string]struct{}
-	ioShim        *common.IOShim
+	iMgrCfg    *IPSetManagerCfg
+	setMap     map[string]*IPSet
+	dirtyCache dirtyCacheMaintainer
```

Comment:
pointer for dirtycache.

Reply:
as Evan mentioned, maps are pointers

Reply:
@vakalapa does it feel like [we're bullying](https://c.tenor.com/t6V-MIkkT8kAAAAC/old-man-how-many-times-do-we-have-to-teach-you.gif) you about this